scalafmt-tests/src/test/resources/rewrite/RedundantBraces-import.stat

```diff
@@ -0,0 +1,32 @@
+rewrite.rules = [RedundantBraces]
```

Comment:
please just append these tests to `RedundantBraces.stat`, we no longer need to create multiple test files since every case can have custom parameters.